### PR TITLE
Adding minimum volume set examples.

### LINKED ISSUE #997: The computeMinimumVolumeLevelSet method has doc issues
See the computeMinimumVolumeInterval method of the Beta:

http://openturns.github.io/openturns/master/user_manual/_generated/openturns.Beta.html?highlight=computeminimumvolumeintervalwithmarginalprobability#openturns.Beta.computeMinimumVolumeInterval

The behaviour of the four methods : 
* computeMinimumVolumeInterval
* computeMinimumVolumeIntervalWithMarginalProbability
* computeMinimumVolumeLevelSet
* computeMinimumVolumeLevelSetWithThreshold

should be clarified within a either a Notebook tutorial or a Reference Guide page with graphics and examples. This would allow to clarify this definition.

### PR DESCRIPTION
**Pull request by mbaudin47**
This PR adds two examples of minimum volume LevelSet:
* in 1D, showing contiguous and non-contiguous cases
* in 2D

This fixes #997.

### CODE REVIEW

**Comment by mbaudin47** [comment]
Good to review.
(The appveyor fails because of pyinstallcheck_InverseNatafIndependentCopulaGradient_std timeout.)

File: ChangeLog [hunk 1]
```diff
@@ -65,6 +65,7 @@
  * #1190 (computeProbability clamped by Domain-SmallVolume)
  * #1202 (Sample::sort & Sample::sortAccordingToAComponent only return new Samples)
  * #1216 (The CalibrationResult doc does not match the code)
+ * #1236 (Adding minimum volume set examples)
```
Comment (jschueller) [inline ChangeLog]: should be `#997 (The computeMinimumVolumeLevelSet method has doc issues)`, see issues/997

Comment (mbaudin47) [reply ChangeLog]: You are right: 53442fc fixes this.

**Comment by mbaudin47** [comment]
This is appveyor:
```
The following tests FAILED:
	335 - pyinstallcheck_StudentFactory_std (Timeout)
```

**Pull request merged**